### PR TITLE
#30 member overview

### PR DESCRIPTION
**Pull request by emanuelen5**
# Changes
Adding a menu specifically for key handouts.

# Demo
Showing how to use it

# Todo
* [X] Extra fields
* [x] Pending orders

### CODE REVIEW

File: admin/src/Membership/MemberBoxOverview.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//         );
+//     }
+// }
+//
+// MemberBoxOverview.contextTypes = {
+//     member: PropTypes.instanceOf(Member)
+// };
+//
+//
+// export default MemberBoxOverview;
```
Comment (andersroos) [inline admin/src/Membership/MemberBoxOverview.jsx:23]: Remove file if unused.

**Review by andersroos — APPROVED** [review]

**Comment by emanuelen5** [comment]
New version.

## Changes
* Same save-button for both key and other fields
* Adding fields for email and phone number
* Some bug fixes

## Demonstration
Animation

**Comment by emanuelen5** [comment]
## Changes
* Spans visualized
* Added zipcode

## Screenshot
image

**Comment by emanuelen5** [comment]
## Changes
* Improved margins (left of sections, and between paragraphs)
* Fixed alignment of some input fields
* Fixed formatting of input fields to look like others

## Screenshot
image

**Comment by emanuelen5** [comment]
Closes #30

**Pull request merged**